contact_manager/contact_manger_linked.c

```diff
@@ -0,0 +1,602 @@
+#include <stdio.h>
+#include <malloc.h>
+#include <unistd.h>
+#include <assert.h>
+
+/**
+ * @file
+ * @implementing a contact manager using linked list inc

```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂